lib/ansible/plugins/action/reboot.py

```diff
@@ -96,7 +106,13 @@ def construct_command(self):
     def get_system_boot_time(self):
         stdout = b''
         stderr = b''
-        command_result = self._low_level_execute_command(self.DEFAULT_BOOT_TIME_COMMAND, sudoable=self.DEFAULT_SUDOABLE)
+
+        # Determine the system distribution in order to use the correct shutdown command arguments
+        uname_result = self._low_level_execute_command('uname')
+        distribution = uname_result['stdout'].strip().lower()
+
+        boot_time_command = self.BOOT_TIME_COMMANDS.get(distribution, self.BOOT_TIME_COMMANDS['linux'])
```

Comment:
Change the default to `self.DEFAULT_BOOT_TIME_COMMAND` instead of `self.BOOT_TIME_COMMANDS['linux']`.